### PR TITLE
fix(li): type fix

### PR DESCRIPTION
**Pull request by Esorakouki**
### PR includes

type fix

### CODE REVIEW

File: packages/lite-insight/src/pipeline/extract.ts [hunk 1]
```diff
@@ -90,10 +90,10 @@ export const extractInsightsFor1M1DCombination = (
 ): InsightInfo<PatternInfo>[][] => {
   const { fieldPropsMap } = referenceInfo;
 
-  const insights: InsightInfo<PatternInfo>[] = [];
+  const insights: InsightInfo<PatternInfo>[][] = [];
 
   dimensions.forEach((dimension) => {
-    const insightsPerDim = [];
+    const insightsPerDim: InsightInfo<PatternInfo>[] = [];
     const collectionForDimension = [];
```
Comment (pddpd) [inline packages/lite-insight/src/pipeline/extract.ts:97]: Should add a type for it?

**Comment by neoddish** [comment]
@Esorakouki Unresolved conversations

File: packages/lite-insight/src/pipeline/extract.ts (hunk 1 above)
Comment (Esorakouki) [reply packages/lite-insight/src/pipeline/extract.ts]: Actually it's a useless const...

**Review by neoddish — APPROVED** [review]
LGTM

**Pull request merged**